proto/osmosis/gamm/v1beta1/query.proto

```diff
@@ -90,9 +92,7 @@ message QueryNumPoolsResponse {
 message QueryPoolParamsRequest {
   uint64 poolId = 1 [ (gogoproto.moretags) = "yaml:\"pool_id\"" ];
 }
-message QueryPoolParamsResponse {
-  oneof params { BalancerPoolParams balancerPoolParams = 1; }
-}
+message QueryPoolParamsResponse { google.protobuf.Any params = 1; }
```

Comment:
I'm a bit worried this will end up causing lots of complexities unless every params contains a `type` field. Perhaps we can add that to every pools query response?

Reply:
Are you referring to `type` field of a string to indicate which type it is?
ex)
```
message QueryPoolParamsResponse {
      google.protobuf.Any params = 1; 
      string poolType = 2;
}
```

Reply:
We discussed in call, `any` already has an `@type` field in the response that makes this easy to parse.